.github/workflows/ci.yml

```diff
@@ -17,9 +17,14 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.7, 3.8, 3.9]
-        django-version: 
+        django-version:
           - django~=3.0.0
           - django~=3.1.0
+          - django~=3.2.0
+        include:
+          - django-version: django~=4.0a1
```

Comment:
```suggestion
          - django-version: django~=4.0b1
```

Reply:
Good catch, but we should now test with django~=4.0rc1